### DISCUSSION TITLE
Suggested addition: `base:` filter for filtering pull requests on their base branch in view filters and the auto-add to project filter.

### DISCUSSION THREAD

**Discussion by Zimmi48** [community/community#53215] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

The `base:` filter has been available for a while in issue search. It would be a nice addition to also include it in the new GitHub projects (both in view filters and in the auto-add to project filter).

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by sbhavani** [comment]
+1 there should be equivalent filters between github issue search and the projects view
